gio/src/auto/enums.rs

```diff
@@ -213,6 +213,8 @@ pub enum CredentialsType {
     NetbsdUnpcbid,
     #[doc(alias = "G_CREDENTIALS_TYPE_APPLE_XUCRED")]
     AppleXucred,
+    #[doc(alias = "G_CREDENTIALS_TYPE_WIN32_PID")]
```

Comment:
it needs a version guard

Reply:
Indeed, it's getting boring. In the next days then :)

Reply:
Done. Anything else @bilelmoussaoui ?